EventEnabledInsurance/prereqs.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+#******************************************************************************
+# Licensed Materials - Property of IBM
+# (c) Copyright IBM Corporation 2019. All Rights Reserved.
+#
+# Note to U.S. Government Users Restricted Rights:
+# Use, duplication or disclosure restricted by GSA ADP Schedule
+# Contract with IBM Corp.
+#******************************************************************************
+
+# PREREQUISITES:
+#   - Logged into cluster on the OC CLI (https://docs.openshift.com/container-platform/4.4/cli_reference/openshift_cli/getting-started-cli.html)
+#
+# PARAMETERS:
+#   -n : <namespace> (string), Defaults to 'cp4i'
+#
+#   With defaults values
+#     ./prereqs.sh
+#
+#   With overridden values
+#     ./prereqs.sh -n <namespace>
+
+function usage() {
+  echo "Usage: $0 -n <namespace>"
+  exit 1
+}
+
+namespace="cp4i"
+tick="\xE2\x9C\x85"
+cross="\xE2\x9D\x8C"
+all_done="\xF0\x9F\x92\xAF"
+SUFFIX="eei"
+POSTGRES_NAMESPACE="postgres"
+
+while getopts "n:r:" opt; do
```

Comment:
```suggestion
while getopts "n:" opt; do
```
`r` option is not used